src/arrayMethodSort.js

```diff
@@ -4,8 +4,17 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (a, b) =>
+    String(a) > String(b)) {
```

Comment:
Better move this function to variable and use it only name

const defaultCompareFunction = (a, b) => String(a) > String(b);

```suggestion
  [].__proto__.sort2 = function(compareFunction = defaultCompareFunction) {
```